src/languages/tsx.coffee

```diff
@@ -18,4 +18,6 @@ module.exports = {
     "tsx"
   ]
 
+  defaultBeautifier: "Pretty Diff"
```

Comment:
I am always concerned with switching defaults.

Reply:
tsx is new and not user facing yet, so it shouldn't be a breaking change.